src/python/database.py

```
@@ -1,9 +1,13 @@
 import sqlite3
 import os
-import logging
 import json
 import uuid
+import logging
+from logging_config import setup_logging, log_execution_time, log_context
+from typing import Any, List, Optional, Tuple, Union
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import

(F401)